x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -7,36 +7,41 @@ import React, { useState } from 'react';
 
 import { i18n } from '@kbn/i18n';
 import { useValues } from 'kea';
-import { EuiSpacer, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiButton, EuiSpacer, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 // @ts-expect-error types are not available for this package yet;
-import { SearchProvider, SearchBox, Sorting } from '@elastic/react-search-ui';
+import { SearchProvider, SearchBox, Sorting, Facet } from '@elastic/react-search-ui';
 // @ts-expect-error types are not available for this package yet
 import AppSearchAPIConnector from '@elastic/search-ui-app-search-connector';
 
 import './search_experience.scss';
 
-import { EngineLogic } from '../../engine';
 import { externalUrl } from '../../../../shared/enterprise_search_url';
 import { useLocalStorage } from '../../../../shared/use_local_storage';
+import { EngineLogic } from '../../engine';
 
-import { SearchBoxView, SortingView } from './views';
+import { Fields, SortOption } from './types';
+import { SearchBoxView, SortingView, MultiCheckboxFacetsView } from './views';
 import { SearchExperienceContent } from './search_experience_content';
 import { buildSearchUIConfig } from './build_search_ui_config';
 import { CustomizationCallout } from './customization_callout';
 import { CustomizationModal } from './customization_modal';
+import { buildSortOptions } from './build_sort_options';
+import { ASCENDING, DESCENDING } from './constants';
 
-const DEFAULT_SORT_OPTIONS = [
+const RECENTLY_UPLOADED = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.documents.search.sortBy.option.recentlyUploaded',
+  {
+    defaultMessage: 'Recently Uploaded',
+  }
+);
```

Comment:
Haha, I think I'm starting to see Scotty's point about pulling i18n strings to separate constants files 🤔 I think this one could be moved without losing any context, totally up to you though